examples/heart_attack_predictor/log_model.py

```diff
@@ -0,0 +1,53 @@
+# Importing Libraries
```

Comment:
How does this example differ from the existing sklearn examples?

Reply:
The existing examples are good but when I was exploring MLflow, I got stuck at many points like
How to log a pre-trained model (all the existing examples have added the training code with it)
How to add signature in your model
How to register your model through code
How to shift your model into production through code

So this example clarifies all the above points